termwiz/src/escape/esc.rs

```diff
@@ -68,10 +68,19 @@ pub enum EscCode {
     /// DECPNM - Normal Keypad
     DecNormalKeyPad = esc!('>'),
 
-    /// Designate Character Set – DEC Line Drawing
-    DecLineDrawing = esc!('(', '0'),
-    /// Designate Character Set – US ASCII
-    AsciiCharacterSet = esc!('(', 'B'),
```

Comment:
There are a couple of tests in the termwiz crate that need to be adjusted for this change:
https://github.com/wez/wezterm/blob/main/termwiz/src/render/terminfo.rs#L891

You can run those tests using `cargo test -p termwiz` (or just run all of the tests: `cargo test`)